vendor.conf

```diff
@@ -95,7 +95,7 @@ github.com/philhofer/fwd 98c11a7a6ec829d672b03833c3d69a7fae1ca972
 github.com/tinylib/msgp 3b556c64540842d4f82967be066a7f7fffc3adad
 
 # fsnotify
-github.com/fsnotify/fsnotify v1.4.7
+github.com/fsnotify/fsnotify c9e9bfb647855178ec5f3947c02e6bd47a379eb9 https://github.com/kolyshkin/fsnotify/
```

Comment:
Per https://github.com/fsnotify/fsnotify/issues/183#issuecomment-286456045 the project is looking for new maintainers; perhaps we can assist in that

Reply:
I'll start with a few PRs :)